HPXMLtoOpenStudio/resources/weather.rb

```diff
@@ -308,9 +308,12 @@ def calc_ground_temperatures
     bo = (data.MonthlyAvgDrybulbs.max - data.MonthlyAvgDrybulbs.min) * 0.5
 
     @data.GroundMonthlyTemps = []
+    # The regression coefficients are from L. Xing's simplified design model ground temperatures (Appendix A-3) and the nearest TMY3 station's annual dry bulb.
+    adj_annual_avg_drybulb_c = 0.91 * UnitConversions.convert(data.AnnualAvgDrybulb, 'F', 'C') + 1.82
+    adj_annual_avg_drybulb_f = UnitConversions.convert(adj_annual_avg_drybulb_c, 'C', 'F')
     for i in 0..11
       theta = amon[i] * 24.0
-      @data.GroundMonthlyTemps << UnitConversions.convert(data.AnnualAvgDrybulb - bo * Math::cos(2.0 * Math::PI / p * theta - po - phi) * gm + 460.0, 'R', 'F')
+      @data.GroundMonthlyTemps << UnitConversions.convert(adj_annual_avg_drybulb_f - bo * Math::cos(2.0 * Math::PI / p * theta - po - phi) * gm + 460.0, 'R', 'F')
```

Comment:
@afontani and I plan to:

Leave this. But in `hvac_sizing.rb`, use `adj_annual_avg_drybulb_f` instead of the average of `@data.GroundMonthlyTemps`.

Reply:
![image](https://github.com/NREL/OpenStudio-HPXML/assets/8516790/ce021865-4b92-4f3e-a707-adf42735dfe7)